pandas/core/arrays/datetimelike.py

```diff
@@ -455,6 +455,12 @@ def ravel(self, *args, **kwargs):
         data = self._data.ravel(*args, **kwargs)
         return type(self)(data, dtype=self.dtype)
 
+    @property
+    def T(self):
```

Comment:
Where is this used? In an only datetimelike context?

Reply:
yes